sonar-project.properties

```diff
@@ -1,4 +1,5 @@
 sonar.host.url=https://sonarcloud.io
 sonar.organization=green-code-initiative
 sonar.projectKey=green-code-initiative_creedengo-rules-specifications
-sonar.sources=.
\ No newline at end of file
+sonar.sources=.
+sonar.exclusions=tools/rule-exporter/**
```

Comment:
For now sources of the tool are not analyzed by SonarQube scanner so I have excluded files here
But the idea is to include them asap (enhancement)